docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+  <PropertyGroup>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
+The following example uses the `Condition` attribute to specify fallbacks only for the `netcoreapp2.1` target:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageTargetFallback Condition="'$(TargetFramework)'=='netcoreapp2.1'">$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
+### RuntimeIdentifier
+
+The `RuntimeIdentifier` property lets you specify a single [runtime identifier (RID)](../rid-catalog.md) for the project. The RID enables publishing a self-contained deployment.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RuntimeIdentifier>ubuntu.16.04-x64</RuntimeIdentifier>
+  </PropertyGroup>
+</Project>
+```
+
+Use `RuntimeIdentifiers` (plural) instead if you need to publish for multiple runtimes. However, `RuntimeIdentifier` can provide faster builds when only a single runtime is required.
+
+### RuntimeIdentifiers
+
+The `RuntimeIdentifiers` property lets you specify a semicolon-delimited list of [runtime identifiers (RIDs)](../rid-catalog.md) for the project. RIDs enable publishing self-contained deployments.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RuntimeIdentifiers>win10-x64;osx.10.11-x64;ubuntu.16.04-x64</RuntimeIdentifiers>
+  </PropertyGroup>
+</Project>
+```
+
+### TargetFramework
+
+The `TargetFramework` property specifies the target framework version for the app, which implicitly references a [metapackage](../packages.md#metapackages). For a list of valid target framework monikers, see [Target frameworks in SDK-style projects](../../standard/frameworks.md#supported-target-framework-versions).
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+</Project>
+```
+
+For more information, see [Target frameworks in SDK-style projects](../../standard/frameworks.md).
+
+### TargetFrameworks
+
+Use the `TargetFrameworks` property when you want your app to target multiple platforms. This property is ignored if `TargetFramework` is specified. For a list of valid target framework monikers, see [Target frameworks in SDK-style projects](../../standard/frameworks.md#supported-target-framework-versions).
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
+  </PropertyGroup>
+</Project>
+```
+
+For more information, see [Target frameworks in SDK-style projects](../../standard/frameworks.md).
+
+## Publish properties
+
+### UseAppHost
+
+The `UseAppHost` property was introduced in the 2.1.4 version of the .NET Core SDK. It controls whether or not a native executable is created for a deployment. A native executable is required for self-contained deployments.
```

Comment:
```suggestion
The `UseAppHost` property was introduced in the 2.1.400 version of the .NET Core SDK. It controls whether or not a native executable is created for a deployment. A native executable is required for self-contained deployments.
```

Is this right? 2.1.4 is a runtime version; SDK versions have three-digit third parts. But I didn't check on when UseAppHost was introduced.